### PR TITLE
Updated VS workload manifests to Windows SDK 19041

### PR DESCRIPTION
**Pull request by DamianEdwards**
Update the vs workload manifests to match latest windows sdk version being used.

### CODE REVIEW

**Review by BrennanConroy — APPROVED** [review]

**Review by dougbu — APPROVED** [review]
Please the root .vsconfig file as well

**Comment by dougbu** [comment]
Oops, "please update"

File: src/Installers/Windows/AspNetCoreModule-Setup/build/settings/common.props [hunk 1]
```diff
@@ -14,6 +14,7 @@
   <!-- General properties -->
 
   <PropertyGroup>
+    <!-- If the following line is updated ensure that the /eng/scripts/vs.17.*.json files are updated to include the same version component too. -->
```
Comment (dougbu) [inline src/Installers/Windows/AspNetCoreModule-Setup/build/settings/common.props:17]: This, this addresses half of my internal comment
> Similar lines exist in src\Installers\Windows\AspNetCoreModule-Setup\build\settings\common.props and src\Servers\IIS\AspNetCoreModuleV2\CommonLibTests\CommonLibTests.vcxproj. I think the second one is redundant but, if that one stays, they both should probably have the same comment.

File: src/Installers/Windows/AspNetCoreModule-Setup/build/settings/common.props [hunk 2]
```diff
@@ -14,6 +14,7 @@
...
     <IisOobWinSdkVersion Condition="'$(IisOobWinSdkVersion)' == ''">10.0.18362.0</IisOobWinSdkVersion>
```
Comment (dougbu) [inline src/Installers/Windows/AspNetCoreModule-Setup/build/settings/common.props:18]: anyone know if this line is now incorrect❔

**Pull request merged**